src/GeositeFramework/js/Plugin.js

```diff
@@ -235,8 +235,10 @@ require(['use!Geosite',
                     showValueKey = pluginObject.toolbarName + " showinfographic";
                 if (typeof localStorage[showValueKey] !== 'undefined') {
                     return localStorage[showValueKey] === 'true';
-                }
-                return true;
+                } else {
+					localStorage.setItem(showValueKey,pluginObject.showInfographicOnStart);
+					return localStorage[showValueKey] === 'true';
+				}
```

Comment:
Hi @mapossum. I believe @kdeloach is suggesting that

```js
localStorage.setItem(showValueKey,pluginObject.showInfographicOnStart);
return localStorage[showValueKey] === 'true';
```

can be replaced with 

```
return pluginObject.showInfographicOnStart;
```

I tested out this PR with and without @kdeloach's suggested change, and they are functionally similar. I think setting the local storage variable has no functional effect because it is only used immediately after it is set from the value in the plugin config. Returning the value from the plugin config is a more direct way to achieve the desired change.